test/Test/Toml/Parsing/Unit/Toml.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE PatternSynonyms  #-}
 
 module Test.Toml.Parsing.Unit.Toml where
```

Comment:
I believe that a lot of functions like `makeKey` are redundant now. Could you please add the minimal required explicit export list to this module so it becomes apparent which old utility functions can be removed now?

Reply:
Hey, just to clarify, you mean add an export list to `Toml.Toml.Parsing.Unit.Common`?

Reply:
@Nimor111 Yes, you're right. I missed the fact that now all functions are in the `Common` module. I think `makeKey` function can be removed as well since there's `IsString` instance for `Key` and all usages like `makeKey ["foo", "bar"]` can be replaced with just `"foo.bar"`.

Reply:
This should wrap it up, i'll do the neat-interpolation package refactoring in another PR. :+1:

Reply:
@Nimor111 Thanks for your work! I think that at this stage `neat-interpolation` is not really required. New tests are already clean enough :+1: Also, since we want to test that everything works with trailing whitespaces, there're always will be tests written in the old style, like `"foo   "` (because editors usually remove trailing whitespaces). So no need to use two ways to describe the same thing, we can live with raw strings for now :slightly_smiling_face:

Reply:
Fair enough. :slightly_smiling_face: